.github/workflows/ci.yml

```diff
@@ -218,14 +200,60 @@ jobs:
     - name: Test libf3d examples
       if: matrix.vtk_version != 'v9.0.0' && matrix.vtk_version != 'v9.1.0'
       working-directory: ${{github.workspace}}
+      shell: bash
       run: ctest --test-dir build_examples -C Release -VV
 
-    - name: Validate plugin examples
+    - name: Build, test and install plugin examples
       working-directory: ${{github.workspace}}
+      shell: bash
       run: |
-        cmake -Werror=dev -Werror=deprecated --warn-uninitialized -Df3d_DIR:PATH=$(pwd)/install/lib/cmake/f3d -DCMAKE_PREFIX_PATH:PATH=$(pwd)/dependencies/install/ -DBUILD_TESTING=ON -B build_plugins source/examples/plugins
+        cmake -Werror=dev -Werror=deprecated --warn-uninitialized -Df3d_DIR:PATH=$(pwd)/install/lib/cmake/f3d -DCMAKE_PREFIX_PATH:PATH=$(pwd)/dependencies/install/ -DCMAKE_INSTALL_PREFIX=$(pwd)/install/ -DBUILD_TESTING=ON -B build_plugins source/examples/plugins
         cmake --build build_plugins --config Release
         ctest --test-dir build_plugins -C Release -VV
+        cmake --install build_plugins --config Release
+        cmake --install build_plugins --config Release --component configuration
+
+    - name: Set F3D_PLUGINS_PATH to install
+      working-directory: ${{github.workspace}}
+      shell: bash
+      run: echo "F3D_PLUGINS_PATH=$(pwd)/install/lib" >> $GITHUB_ENV
+
+    - name: Set F3D binary path Linux
+      if: runner.os == 'Linux'
+      shell: bash
+      run: echo "F3D_BIN_PATH=./bin/f3d" >> $GITHUB_ENV
+
+    - name: Set F3D binary path Windows
+      if: runner.os == 'Windows'
+      shell: bash
+      run: echo "F3D_BIN_PATH=./bin/f3d.exe" >> $GITHUB_ENV
```

Comment:
Isn't it supposed to be `./bin/Release/f3d.exe` on Windows?

Reply:
not in the install

Reply:
right!